lib/sync_checker/checks/translations_check.rb

```diff
@@ -0,0 +1,32 @@
+module SyncChecker
+  module Checks
+    TranslationsCheck = Struct.new(:expected_locales) do
+      attr_reader :content_item
+      def call(response)
+        stringified_locales = expected_locales.map(&:to_s)
```

Comment:
Do we care if the `expected_locales` passed to the constructor is `nil` but there are `available_translations` in the response?